TCAT/NotificationToggleTableViewCell.swift

```diff
@@ -68,12 +77,26 @@ class NotificationToggleTableViewCell: UITableViewCell {
         }
     }
 
-    func configure(for notificationTitle: String, isFirst: Bool) {
-        notificationTitleLabel.text = notificationTitle
+    func configure(for type: NotificationType, isFirst: Bool, delegate: NotificationToggleTableViewDelegate? = nil) {
+        self.delegate = delegate
+        self.type = type
+        notificationTitleLabel.text = type.title
         if isFirst {
             setupFirstHairline()
         }
     }
+    
+    @objc func switchValueChanged() {
+        if notificationSwitch.isOn {
+            switch type {
+            case .beforeBoarding:
+                delegate?.displayNotificationBanner(type: .beforeBoardingConfirmation)
+            case .delay:
+                delegate?.displayNotificationBanner(type: .delayConfirmation)
+            default: break
```

Comment:
does `default: break` actually do anything? or can that be removed? or is it because `switch` statements in swift have to be exhaustive?

Reply:
yep. i need to keep the `default` so it's exhaustive